catalog/spatial/registry/registry-schema-bindings/src/test/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelperTest.java

```diff
@@ -51,6 +64,13 @@ public void testGetString() throws Exception {
         InternationalStringType ist = getTestInternationalStringType();
 
         String istString = istHelper.getString(ist);
+
+        if (!isDefaultLocaleInTestSet()) {
```

Comment:
Updated tests to split default Locale into a separate test and remove the conditional